WordPress/Classes/ViewRelated/System/WPTabBarController+QuickStart.swift

```diff
@@ -10,20 +10,16 @@ extension WPTabBarController {
 
             guard let userInfo = notification.userInfo,
                 let element = userInfo[QuickStartTourGuide.notificationElementKey] as? QuickStartTourElement,
-                [.newpost, .readerTab].contains(element),
+                [.readerTab].contains(element),
                 let tabBar = self?.tabBar else {
                     return
             }
 
             let newSpotlight = QuickStartSpotlightView()
             tabBar.addSubview(newSpotlight)
 
-            let x: CGFloat
-            if element == .newpost {
-                x = tabBar.bounds.size.width / 2.0
-            } else {
-                x = tabBar.bounds.size.width * 0.40 - newSpotlight.frame.size.width
-            }
+            let x = tabBar.bounds.size.width / 2.0
```

Comment:
I was going to comment that the spotlight isn't completely centered on the Reader tab, but actually I think I prefer this otherwise it would obscure the icon.